Added another example to illustrate use for only_if (#5269)
chef/inspec: docs-chef-io/content/inspec/dsl_inspec.md

end
```

> Here is another simple example, which checks for, if certain pip packages are installed, only if /root/.aws exists

Reviewer: This is a valid example of `only_if` - I wanted to confirm, is there something about this case that technically differs from the previous example? It looks like (and your description for the example says) this is simply one more example, but I wanted to make sure. If there's something that doesn't work in the previous example that this example solves, we should highlight that.

Author: @james-stocks this additional example, I felt, explains better (better because, at first glance, the example was not explanatory enough to understand how it worked), and also provides an example to check directory exists as a condition.

> Here is another simple example, which checks for, if certain pip packages are installed, only if /root/.aws exists

Reviewer 2: @unixengineer Hello! Please update your PR to include the following wording. Thank you for adding this example!

```suggestion
This example checks for if certain pip packages are installed, but only if '/root/.aws' exists:
```

Author: Will update the PR and add a signed by too

Author: The PR should have the updated changes now.
